homeassistant/components/switch/pilight.py

```diff
@@ -21,13 +22,16 @@
 CONF_ON_CODE_RECIEVE = 'on_code_receive'
 CONF_SYSTEMCODE = 'systemcode'
 CONF_UNIT = 'unit'
+CONF_UNITCODE = 'unitcode'
 
 DEPENDENCIES = ['pilight']
 
-COMMAND_SCHEMA = pilight.RF_CODE_SCHEMA.extend({
+COMMAND_SCHEMA = vol.Schema({
+    vol.Optional(CONF_PROTOCOL): cv.string,
     vol.Optional('on'): cv.positive_int,
     vol.Optional('off'): cv.positive_int,
     vol.Optional(CONF_UNIT): cv.positive_int,
+    vol.Optional(CONF_UNITCODE): cv.positive_int,
```

Comment:
Sorry for the late reply, I did not know that a review can be just pending without submitting. I had it pending for days...